### PR TITLE
feat: added speakers to events route

### LINKED ISSUE #125: Add event speakers to events route (UI only)
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

screen shot 2017-06-04 at 7 41 57 am

Located at `events/<event_id>/speakers`

@niranjan94 can I work on this

### PR DESCRIPTION
**Pull request by utkarshnath**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

screen shot 2017-06-06 at 4 14 25 am

Fixes #125 
https://hidden-badlands-35601.herokuapp.com/events/963d3ba5/speakers
@niranjan94 @geekyd @CosmicCoder96 plz review

### CODE REVIEW

File: app/templates/events/view/speakers.hbs [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+            <th>{{t 'Name'}}</th>
+            <th>{{t 'Email'}}</th>
+            <th>{{t 'Phone'}}</th>
+            <th>{{t 'Submitted Sessions'}}</th>
+            <th>{{t 'State'}}</th>
+            <th></th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
```
Comment (niranjan94) [inline app/templates/events/view/speakers.hbs]: Please load data from a dummy js array.

**Comment by utkarshnath** [comment]
@niranjan94 plz check https://hidden-badlands-35601.herokuapp.com/events/963d3ba5/speakers

File: app/templates/events/view/speakers.hbs [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+            <th>{{t 'State'}}</th>
+            <th></th>
+          </tr>
+        </thead>
+        <tbody>
+          {{#each model as |speaker|}}
+            <tr>
+              <td class="right aligned">
+                <div>
+                  <img class="ui mini avatar round image" alt="speakers_image" src={{speaker.image}}>
```
Comment (geekyd) [inline app/templates/events/view/speakers.hbs]: Apply a check for the image. It can have `''` as value.

Comment (niranjan94) [inline app/templates/events/view/speakers.hbs]: Or use the safe image component. Would automatically load a palceholder when image doesn't exisit.

Comment (utkarshnath) [inline app/templates/events/view/speakers.hbs]: ok

**Comment by utkarshnath** [comment]
@niranjan94 @geekyd plz review
https://hidden-badlands-35601.herokuapp.com/events/963d3ba5/speakers

File: app/templates/events/view/speakers.hbs [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+            <th>{{t 'State'}}</th>
+            <th></th>
+          </tr>
+        </thead>
+        <tbody>
+          {{#each model as |speaker|}}
+            <tr>
+              <td class="right aligned">
+                <div>
+                  <a class="ui avatar round image">
```
Comment (geekyd) [inline app/templates/events/view/speakers.hbs]: @utkarshnath It would look better if we have image inside the name column only.

Comment (utkarshnath) [inline app/templates/events/view/speakers.hbs]: @geekyd updated plz check

**Pull request merged**